### PR TITLE
Nv 2024 issue 2630

### PR DESCRIPTION
**Pull request by nafisa-valieva**
I have tried to update three pages of the guidelines, I hope I have not messed up and my changes are helpful!  Thank you for your insights for improvements! 
(https://github.com/BetaMasaheft/Documentation/issues/2699, https://github.com/BetaMasaheft/Documentation/issues/2690, https://github.com/BetaMasaheft/Documentation/issues/714)

### CODE REVIEW

File: pages/Manuscripts.xml
Comment (thea-m) [inline pages/Manuscripts.xml:74]: Including references to issues is rarely done in the guidelines so far. On the one hand, this is very helpful and improves transparency, on the other, it makes the text somewhat more complex and longer. Maybe a compromise could be to move the reference into paranthesis at the end of the paragraph ("(see issue XX)"), so that it can be skipped easier when skimming the guidelines.

File: pages/Manuscripts.xml
Comment (thea-m) [inline pages/Manuscripts.xml:77]: quotations marks around "initial version", I think

File: pages/Manuscripts.xml
Comment (thea-m) [inline pages/Manuscripts.xml:76]: I think it would be nice to include as an example the complete `<revisionDesc>` of a file that followed this process (as two `<egXML xmlns="http://www.tei-c.org/ns/Examples"> `elements). The original one using "stub" could be included after "one needs to write the word "stub" anywhere in the change history.", and the revised one after "initial version or anything else". ANLcr1 is a real example that comes to my mind just because I was involved, any other file would work as well.

File: pages/Manuscripts.xml
Comment (thea-m) [inline pages/Manuscripts.xml:78]: I would leave out "on the opposite", because there is no argumentation here, just a description of two different cases

File: pages/Manuscripts.xml
Comment (thea-m) [inline pages/Manuscripts.xml:81]: I think here as well a real example from a record would be helpful

File: pages/bibliographic-references.xml
Comment (thea-m) [inline pages/bibliographic-references.xml:61]: I think there is an English problem in "one can include the marked-up by a", but don't spontaneously see a good correction :)

File: pages/bibliographic-references.xml
Comment (thea-m) [inline pages/bibliographic-references.xml:73]: "the visualisation of the record on the app"

File: pages/bibliographic-references.xml
Comment (thea-m) [inline pages/bibliographic-references.xml:79]: I think here the better markup would be `<tag><bibl><ptr target="bm:BmWebsite"/></bibl></tag>` for the entire tag

File: pages/workHeader.xml
Comment (thea-m) [inline pages/workHeader.xml:190]: I think I would delete "if one click 'text' in the upper menu", because this is secondary.

File: pages/workHeader.xml
Comment (thea-m) [inline pages/workHeader.xml:199]: acknowledged

File: pages/workHeader.xml
Comment (thea-m) [inline pages/workHeader.xml:201]: Is there an example of this use of `<note>` that could be included in the following example?

File: pages/workHeader.xml
Comment (thea-m) [inline pages/workHeader.xml:269]: marked up

File: pages/workHeader.xml
Comment (thea-m) [inline pages/workHeader.xml:271]: My impression is that "and also might contain manuscripts which we do not have yet" is now in contradiction to the above described practice of creating stub records. Maybe it could be deleted.

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, this is a great improvement and very helpful. I have made many comments only due to the complexity of the topic.

File: pages/Manuscripts.xml
Comment (eu-genia) [reply pages/Manuscripts.xml:74]: Link to the issue must be included in the description of the pull request, then it can always be found (not as title but as a link, or several links - you can edit the very first message in this pull request)

For better documentation it is always better to have each issue clearly linked to each change in github history, so it is not ideal to have several files for several issues in one pull request (we are not going to change it now, but for the future).

Please remove the links to the issues from the guidelines, this is not something we want.

Comment (nafisa-valieva) [reply pages/Manuscripts.xml:74]: @eu-genia @thea-m I have followed the example for issue 368 (https://github.com/BetaMasaheft/Documentation/issues/368)
in https://betamasaheft.eu/Guidelines/?id=manuscripts
Is it a different case or should I change that part too?

File: pages/Manuscripts.xml
Comment (nafisa-valieva) [reply pages/Manuscripts.xml:81]: Here I refer to a different page óf the guidelines. Should I rather repeat everything?

Comment (thea-m) [reply pages/Manuscripts.xml:81]: ah, I missed this, then it should be enough like this!

File: pages/workHeader.xml
Comment (thea-m) [reply pages/workHeader.xml:201]: Then maybe better just leave it, I think

File: pages/bibliographic-references.xml
Comment (thea-m) [reply pages/bibliographic-references.xml:62]: Would "one can include a <gi>bibl</gi> element" work?

Comment (nafisa-valieva) [reply pages/bibliographic-references.xml:62]: fine for me!

**Review by thea-m — APPROVED** [review]
Wonderful, thank you!

**Comment by nafisa-valieva** [comment]
@eu-genia, I have a question: see that technically merging is possible. Do I need to wait for your active approval, or is Dorothea´s review enough to merge the updates?

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:88]: Differentiating

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:97]: I would remove the sentence

"After the description is
               completed, the above example can be changed into something like the example below."


It repeats what has been said and misleads, as this change is necessary not only when the description has been **completed**, but at any point when it is not a stub any more (the judgement on that is up to the cataloguer of course)

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:95]: must **be** replaced

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:96]: "something like "initial version" or anything else" - does not sound good as a guideline. Guidelines should be clear and concise. I think `"the word "stub" in the <gi>change</gi> element must be replaced" `is sufficient.

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:27]: Differentiating

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:89]: either
if the cataloguer wants **that** others consider
or
if the cataloguer wants others **to** consider

(you can also use chatGPT for polishing English)

File: pages/Manuscripts.xml
Comment (eu-genia) [inline pages/Manuscripts.xml:90]: as **a** "stub"

**Review by eu-genia — CHANGES_REQUESTED** [review]
Thank you.

Please as already suggested -  in the very first comment in this pull request add links to all the issues and discussions these changes attend to. Then the commit will be linked to the issues and one will be able to properly document what has been done.

Thank you.

**Comment by eu-genia** [comment]
> @eu-genia, I have a question: i see that technically merging is possible. Do I need to wait for your active approval, or is Dorothea´s review enough to merge the updates?

I think we have talked about this already.

File: pages/bibliographic-references.xml
Comment (nafisa-valieva) [reply pages/bibliographic-references.xml]: > I think here the better markup would be `<tag><bibl><ptr target="bm:BmWebsite"/></bibl></tag>` for the entire tag

the schema does not allow it.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**
